### PR TITLE
JumpToStart

### PR DESCRIPTION
**Pull request by cheesesamwich**
A fixed up version of https://github.com/Vendicated/Vencord/pull/2171, I deleted my first fork a while ago so i need to make a new PR- Hopefully the formatting is more consistent now, i also tested it with all the cases i could think of:

- DMs
- Standard Channels
- Forums
- Voice Chats
- Hidden Channels (Just does nothing)
- Threads

### CODE REVIEW

**Comment by Vendicated** [comment]
seems useful! perhaps it might be useful to add a "Jump to first message" button somewhere here in forums?

i keep wanting to jump to the first forum post and wondering why the fuck discord hasn't added an intuitive way to do so

image

**Comment by cheesesamwich** [comment]
> seems useful! perhaps it might be nice to add a "Jump to first message" button somewhere here in forums?
> 
> i keep wanting to jump to the first forum post and wondering why the fuck discord hasn't added an intuitive way to do so
> 
> image

sounds good, i'll start working on it right now

File: src/plugins/jumpToStart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            id="vc-jump-to-first"
+            label="Jump To First Message"
+            action={() => NavigationRouter.transitionTo(`/channels/${channel.guild_id ?? "@me"}/${channel.id}/0`)}
+        />
+    );
+};
+
+export default definePlugin({
+    name: "JumpToStart",
+    description: "Adds a context menu option to jump to the start of a channel/DM",
```
Comment (Vendicated) [inline src/plugins/jumpToStart/index.tsx:26]: let's do this for easier discoverability
```suggestion
    description: "Adds a context menu option to jump to the first message of channel/DM",
    tags: ["JumpToFirst", "FirstMessage"],
```

**Review by Vendicated — COMMENTED** [review]
seems to work well. please make sure to add a README.md!

**Comment by cheesesamwich** [comment]
Sorry for the inactivity, i kinda sorta forgot. Working on the forum header button now

**Pull request closed without merging**